src/logger.test.ts

```diff
@@ -164,20 +165,22 @@ test('Optional tag', () => {
   lc.debug?.('a');
   expect(mockDebug.lastCall.args).to.deep.equal(['a']);
 
-  const lc2 = lc.addContext('b');
+  const lc2 = lc.withContext('b');
   lc2.debug?.('c');
   expect(mockDebug.lastCall.args).to.deep.equal(['b', 'c']);
 
-  const lc3 = lc.addContext('d', 'e');
+  const lc3 = lc.withContext('d', 'e');
   lc3.debug?.('f');
   expect(mockDebug.lastCall.args).to.deep.equal(['d=e', 'f']);
 });
 
 class TestLogSink implements LogSink {
-  messages: [LogLevel, ...unknown[]][] = [];
+  // Note: Order in messages is different from log args order to verify
```

Comment:
You could also have used:

```ts
  messages: [LogLevel, Context | undefined, unknown[]][] = [];
```

Reply:
Love it. Now my verification code reads as it should:

```js
    ['info', {abc: 'is', easy: 'as'}, [1, 2, 3]],
```

Thanks!